### PR TITLE
Add missing `Column Types` docs

### PR DESCRIPTION
**Pull request by LouisAUTHIE**
Adds color Column link

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Adds colorcolumn link to get to colorcolumn doc
| Fixed ticket? | N/A

### CODE REVIEW

File: development/components/grid/columns-reference/color.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<?php
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\ColorColumn;
+use PrestaShop\PrestaShop\Core\Grid\Column\ColumnCollection;
+
+$colorColumn = new ColorColumn('color');
+$dataColumn->setName('color');
+$dataColumn->setOptions([
+     'field' => 'color',
+     'color_field' => 'color'
+]);
```
Comment (PululuK) [inline development/components/grid/columns-reference/color.md]:
```suggestion
$colorColumn = new ColorColumn('color');
$colorColumn->setName('color');
$colorColumn->setOptions([
     'field' => 'color',
     'color_field' => 'color'
]);
```

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @LouisAUTHIE

**Comment by matks** [comment]
@LouisAUTHIE Would you submit a PR to add this to branch `1.7.x` ?

**Comment by LouisAUTHIE** [comment]
> @LouisAUTHIE Would you submit a PR to add this to branch `1.7.x` ?

Hi thanks @matks , he is the new branch for the 1.7.x doc https://github.com/PrestaShop/docs/pull/1384

**Comment by thomasnares** [comment]
@kpodemski already backported, please remove label

**Pull request merged**